src/routes/User.jsx

```diff
@@ -0,0 +1,95 @@
+// React Imputs
+import React, { useContext, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+// Custom Hook Imports
+import { useStatusNotification } from '../hooks';
+// Utility Imports
+import { getDocTTLs, runNotification } from '../utils';
+// Context Imports
+import { SignedInUserContext } from '../contexts';
+// Component Imports
+import { ShowDocumentsModal } from '../components/DocumentModals';
+import FormSection from '../components/Form/FormSection';
+
+/**
+ * User - Page that displays the User's stored Documents
+ *
+ * @memberof Pages
+ * @name User
+ * @returns {React.ReactElement} The user page
+ */
+
+const User = () => {
+  const location = useLocation();
+  localStorage.setItem('restorePath', location.pathname);
+
+  const { state, dispatch } = useStatusNotification();
+
+  // EXPERIMENTAL - Event handler for displaying Documents from PASS
```

Comment:
seems to be extremely similar to function within `ClientListTableRow.jsx`, could we refactor/pull out and make work for both?

Reply:
Yeah, think that would be good. I'll refactor these

Reply:
Factored

Reply:
Removing function in favor of an updated getDocuments function